song/values.yaml

```diff
@@ -69,6 +69,13 @@ appConfig:
     jwt:
       enable: false
       publicKeyUrl: https://auth.local/oauth/public_key
+    score:
+      url: https://storage.local
+    clientCredentials:
```

Comment:
in addition, we want this to be backwards compatible, so something like this would make sense
```yaml
score:
  url: https://score.example.org
  auth:
     useApikey: "true"
     apiKey:  my-api-key-with-score.WRITE-scope
     clientCredentials:
        id: oauth
        secret: oauthsercret
        tokenUrl : blah
        systemScope: score.WRITE
```